tests/Directory.Build.targets

```diff
@@ -0,0 +1,12 @@
+<Project>
+
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)../'))" />
+
+  <ItemGroup Condition="'$(UnitTestType)' == 'nunit'">
+    <!-- <PackageReference Include="Microsoft.NET.Test.Sdk" Version="$(MicrosoftNETTestSdkPackageVersion)" /> -->
+    <PackageReference Include="NUnit" Version="$(NUnitPackageVersion)" />
+    <PackageReference Include="NUnit3TestAdapter" Version="$(NUnit3TestAdapterPackageVersion)" />
+    <PackageReference Include="NunitXml.TestLogger" Version="$(NunitXmlTestLoggerPackageVersion)" />
```

Comment:
Do you need NUnit support in Arcade?

Reply:
Probably not worth the effort, we're slowly converting to xunit anyways.

Reply:
OK, Just curious. There are a couple of other repos that needed it.